src/components/Filter/Filter.js

```diff
@@ -0,0 +1,44 @@
+export function Filter(){
+    return(
+        <aside className="filter-sidenav">
+                <div className="f-title fw-bold"> 
+                    <span>Filters</span>
+                    <span className="clr-filter">Clear</span>
+                </div>
+
+                <form action="" className="filter-form">
+                    <label htmlFor="price" className="fs-md fw-semibold">
+                        Price Range
+                    </label>
+                    <input type="range" name="price" id="" />
+
+                    <label htmlFor="category" className="fs-md fw-semibold">
+                        Category:
+                    </label>
+                    <label htmlFor="rating"><input type="checkbox" name="rating" id="" />T-Shirts</label>
+                    <label htmlFor="rating"><input type="checkbox" name="rating" id="" />Formal Shirts</label>
+                    <label htmlFor="rating"><input type="checkbox" name="rating" id="" />Casual Shirts</label>
+                    <label htmlFor="rating"><input type="checkbox" name="rating" id="" />Jeans</label>
+                    <label htmlFor="rating"><input type="checkbox" name="rating" id="" />Trousers</label>
+
+                    <label htmlFor="rating" className="fs-md fw-semibold">
+                            Rating
+                    </label>
+
+                    <label htmlFor="rating"><input type="radio" name="rating" id="" />4 Stars & above</label>
```

Comment:
provide value attribute to your inputs. Also the id should be given the value of htmlFor of respective labels.